{"uid":"d40e2f5ab619eaa","name":"test_restart","fullName":"tests.integration.ha_tests.test_restart#test_restart","historyId":"cfa02b8e0221b13995c1b33931e2d886","time":{"start":1760579290537,"stop":1760579290537,"duration":0},"description":"Test restart of all the units simultaneously.","descriptionHtml":"<p>Test restart of all the units simultaneously.</p>\n","status":"skipped","statusMessage":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 720, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 549, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 640, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1760578279969,"stop":1760578279969,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1760578279969,"stop":1760578280255,"duration":286},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1760578279969,"stop":1760578279969,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1760579290537,"stop":1760579290537,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Test restart of all the units simultaneously.","status":"skipped","statusMessage":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 720, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 549, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 640, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"4770e4d7c892030a","name":"log","source":"4770e4d7c892030a.txt","type":"text/plain","size":1130}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1760579290551,"stop":1760579290966,"duration":415},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1760579290966,"stop":1760579290966,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1760579290966,"stop":1760579290967,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1760579290967,"stop":1760579290967,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_restart"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"21508-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_restart"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"f2e9c25c8abe45e2","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":1,"broken":0,"skipped":2,"passed":335,"unknown":37,"total":375},"items":[{"uid":"33725ec7d9c7f901","reportUrl":"../4654//#testresult/33725ec7d9c7f901","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1760492867738,"stop":1760492867738,"duration":0}},{"uid":"8fc14ccbaaa9938a","reportUrl":"../4646//#testresult/8fc14ccbaaa9938a","status":"passed","time":{"start":1760405598153,"stop":1760405773691,"duration":175538}},{"uid":"9f72fdf63a02175","reportUrl":"../4643//#testresult/9f72fdf63a02175","status":"passed","time":{"start":1760319272866,"stop":1760319469408,"duration":196542}},{"uid":"efbedcc3ec50afcc","reportUrl":"../4642//#testresult/efbedcc3ec50afcc","status":"passed","time":{"start":1760232816488,"stop":1760233006062,"duration":189574}},{"uid":"e0d52f5c05e8c5d7","reportUrl":"../4640//#testresult/e0d52f5c05e8c5d7","status":"passed","time":{"start":1760146458209,"stop":1760146650214,"duration":192005}},{"uid":"f47b6048424f8f4e","reportUrl":"../4638//#testresult/f47b6048424f8f4e","status":"passed","time":{"start":1760060010840,"stop":1760060206314,"duration":195474}},{"uid":"ad6b8b7f51a3446e","reportUrl":"../4637//#testresult/ad6b8b7f51a3446e","status":"passed","time":{"start":1759973625634,"stop":1759973799429,"duration":173795}},{"uid":"ed449537e72aa3ab","reportUrl":"../4634//#testresult/ed449537e72aa3ab","status":"passed","time":{"start":1759887264376,"stop":1759887441838,"duration":177462}},{"uid":"6e1962e036c2342b","reportUrl":"../4627//#testresult/6e1962e036c2342b","status":"passed","time":{"start":1759800848025,"stop":1759801052928,"duration":204903}},{"uid":"1006632b6d7d62e5","reportUrl":"../4621//#testresult/1006632b6d7d62e5","status":"passed","time":{"start":1759714423999,"stop":1759714614103,"duration":190104}},{"uid":"b16e1342fdb9b632","reportUrl":"../4620//#testresult/b16e1342fdb9b632","status":"passed","time":{"start":1759628089751,"stop":1759628286917,"duration":197166}},{"uid":"b8b0f1d1efd9c624","reportUrl":"../4618//#testresult/b8b0f1d1efd9c624","status":"passed","time":{"start":1759541646372,"stop":1759541848754,"duration":202382}},{"uid":"215de303962f3c88","reportUrl":"../4615//#testresult/215de303962f3c88","status":"passed","time":{"start":1759455221471,"stop":1759455419289,"duration":197818}},{"uid":"13f3c075571873df","reportUrl":"../4608//#testresult/13f3c075571873df","status":"passed","time":{"start":1759368950686,"stop":1759369382603,"duration":431917}},{"uid":"d6e37cb19f885169","reportUrl":"../4592//#testresult/d6e37cb19f885169","status":"passed","time":{"start":1759282616731,"stop":1759282997707,"duration":380976}},{"uid":"de422626fa89bc16","reportUrl":"../4582//#testresult/de422626fa89bc16","status":"passed","time":{"start":1759197261224,"stop":1759197676114,"duration":414890}},{"uid":"1c2b32a276550177","reportUrl":"../4572//#testresult/1c2b32a276550177","status":"passed","time":{"start":1759109928793,"stop":1759110330830,"duration":402037}},{"uid":"9d809a941b3bd939","reportUrl":"../4571//#testresult/9d809a941b3bd939","status":"passed","time":{"start":1759023986738,"stop":1759024368369,"duration":381631}},{"uid":"8e552f2be520ef31","reportUrl":"../4570//#testresult/8e552f2be520ef31","status":"passed","time":{"start":1758937030402,"stop":1758937441380,"duration":410978}},{"uid":"1bb52bb67f68c315","reportUrl":"../4563//#testresult/1bb52bb67f68c315","status":"passed","time":{"start":1758850664557,"stop":1758851069434,"duration":404877}}]},"tags":["asyncio","abort_on_fail"]},"source":"d40e2f5ab619eaa.json","parameterValues":[]}